packages/package-store/src/storeController/prune.ts

```diff
@@ -14,7 +14,9 @@ export default async function prune (storeDir: string) {
   globalInfo('Removed all cached metadata files')
   const pkgIndexFiles = [] as string[]
   const removedHashes = new Set<string>()
-  const dirs = await fs.readdir(cafsDir)
+  const dirs = (await fs.readdir(cafsDir, { withFileTypes: true }))
+    .filter(dir => dir.isDirectory)
```

Comment:
isDirectory is a function, I believe https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_dirent_isdirectory

```suggestion
    .filter(dir => dir.isDirectory())
```